### DISCUSSION TITLE
Hello everyone.

### DISCUSSION THREAD

**Discussion by NeptuneUlt** [community/community#131197] [New to GitHub]
Where is a good place to start my education on GitHub?

**Comment by gaomingzhao666** [comment]
Hey bro.

**`GitHub Explore` seems like a good place where you can find some awesome projects to start your education.** 

**You can exploit the `Explore` `Topic` and `Trending` panels to filter the characteristics that you prefer.**
- The Explore panel prefers to find projects according to your activities on GitHub.
- The Topic panel is used to find projects according to the languages or frameworks you selected.
- The Trending panel prefers to find projects depending on the recent tendencies.

If this content can answer your question, please mark this comment as an answer. 😊

**Comment by samucodesh** [comment]
> Where is a good place to start my education on GitHub?

Hello @NeptuneUlt 

https://learn.microsoft.com/en-us/collections/p0ygh4erdrmkp7

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @NeptuneUlt 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by ulyssecoispellier** [comment]
You should follow this video for freecodecamp.org it's really clear and comprehensive.

https://www.youtube.com/watch?v=RGOj5yH7evk
